docs/tutorials/losses_contrastive.ipynb

```diff
@@ -0,0 +1,236 @@
+{
```

Comment:
<p>This block is no longer needed, colab is using 2.x by default :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/1732/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>